### DISCUSSION TITLE
Problem launching Benny Launcher

### DISCUSSION THREAD

**Discussion by leescan** [playbenny/benny#449] [General]
thanks so much for sharing this James, i cant wait to try. When i try opening benny.maxproj , Max launches but thats it and in the Max Console i get this below, forgive me if i'm being thick about it

<img width="649" alt="Screenshot 2024-08-01 at 09 08 04" src="https://github.com/user-attachments/assets/2b294020-9a58-431e-819a-4e4c52842043">

**Comment by leescan** [comment]
actually , i noticed the openactions.txt was set to open with Text Edit. I've changed to open with Max and the Benny Launcher has launched!

**Reply by jamesholdenmusic** [reply]
thanks for reporting it, i've added this to the list of possible install problems and solutions - let me know if i've misexplained your fix?
https://playbenny.github.io/benny_manual/installation/#installation-troubleshooting-faq

**Reply by leescan** [reply]
Yes thats accurate to me
